Source/JavaScriptCore/bytecode/UnlinkedFunctionExecutable.h

```diff
@@ -262,6 +264,7 @@ class UnlinkedFunctionExecutable final : public JSCell {
     unsigned m_lineCount : 31;
     unsigned m_hasCapturedVariables : 1;
     unsigned m_unlinkedFunctionNameStart : 31;
+    unsigned m_implementationVisibility : bitWidthOfImplementationVisibility;
```

Comment:
Can you make a room in this class and put a flag? Right now, this is 96byte, so adding this 1 bit enlarges it by 16bytes. And this is super frequently allocated class.

Let's modify ParserModes.h's LexicalScopeFeatures related codes to make it 3 bits (right now, it is 4 bits), and allocate 1 bit room.

Reply:
Let's move this to next to m_lexicalScopeFeatures to make it packed well.

Reply:
Let's also add

```
#if COMPILER(CLANG) && !ASSERT_ENABLED
static_assert(sizeof(UnlinkedFunctionExecutable) <= 96, "UnlinkedFunctionExecutable needs to be small");
#endif
```

in this file to ensure this critical condition.